Networking/README.md

```diff
@@ -0,0 +1,11 @@
+# Networking
+
+Networking layout for REST Api  
+Async/await and generic by protocols 
```

Comment:
I think a better description will be "Does all the heavy lifting for accessing web APIs, like the emoji list"